ekip/everykid/templates/index.html

```diff
@@ -29,23 +30,35 @@
     <nav class="home-mobile-menu mobile-menu">
       <ul class="menu">
         <li>
-          <a href="{% url 'how_it_works' %}">How It Works</a>
+          <a href="{% url 'how_it_works' %}">{% trans "How It Works" %}</a>
         <li>
-          <a href="{% url 'get_your_pass' %}">Get Your Pass</a>
+          <a href="{% url 'get_your_pass' %}">{% trans "Get Your Pass" %}</a>
         </li>
         <li>
-          <a href="{% url 'plan_your_trip' %}">Plan a Trip</a>
+          <a href="{% url 'plan_your_trip' %}">{% trans "Plan a Trip" %}</a>
         </li>
         <li>
-          <a href="{% url 'parents' %}">Parents</a>
-        </li>
+          <a href="{% url 'parents' %}">{% trans "Parents" %}</a>
+        </li>	
```

Comment:
I misunderstood your comment earlier. Yes, it makes sense to type it out once and then just do an include.